docs/framework/performance/writing-large-responsive-apps.md

```diff
@@ -271,7 +271,7 @@ private static string GetStringAndReleaseBuilder(StringBuilder sb)
  This simple caching strategy adheres to good cache design because it has a size cap.  However, there is more code now than in the original, which means more maintenance costs.  You should adopt the caching strategy only if you’ve found a performance problem, and PerfView has shown that <xref:System.Text.StringBuilder> allocations are a significant contributor.  
   
 ### LINQ and lambdas  
- Using Language-Integrated Query (LINQ) and lambda expressions is a great example of using productive features that you might later find you need to rewrite if the code becomes a significant impact on performance.  
+Language-Integrated Query (LINQ), in conjunction with lambda expressions, is a great example of a productive feature. However, its use may have a significant impact on performance over time, and you might find you need to rewrite your code.
```

Comment:
productive --> productivity
I think that I'd delete "great"; simply an example of a productivity feature

Reply:
Hi @rpetrusha ! Thanks for your remarks. I just pushed a PR to address them.